Neos.Flow/Classes/Error/DebugExceptionHandler.php

```diff
@@ -64,7 +64,8 @@ protected function echoExceptionWeb($exception)
             header(sprintf('HTTP/1.1 %s %s', $statusCode, $statusMessage));
         }
 
-        if (!isset($this->renderingOptions['templatePathAndFilename'])) {
+        $useCustomErrorView = isset($this->renderingOptions['templatePathAndFilename']) || isset($this->renderingOptions['renderingGroup']);
```

Comment:
This could be extracted to a (protected) method of the `AbstractExceptionHandler` so that we could replace it below (and in the `ProductionExceptionHandler`) via:
```php
if (!$this->useCustomErrorView()) {
  ...
}
```

Reply:
Done ;)